<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1015</width>
    <height>749</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Log explorer</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout_6" stretch="0,1">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <item>
       <widget class="QGroupBox" name="groupBox">
        <property name="title">
         <string>Files:</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout" stretch="1,0,0,0,0">
         <item>
          <widget class="QListWidget" name="fileListWidget"/>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QLabel" name="label">
             <property name="text">
              <string>File filter:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="patternEdit">
             <property name="text">
              <string>*.log</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="0,1">
           <item>
            <widget class="QLabel" name="label_2">
             <property name="text">
              <string>Count:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="fileLabel">
             <property name="text">
              <string>-</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_5">
           <item>
            <widget class="QPushButton" name="addFolderButton">
             <property name="text">
              <string>Add files from folder...</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="recursiveCheckBox">
             <property name="text">
              <string>Recursive</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QPushButton" name="clearAllPushButton">
           <property name="text">
            <string>Clear all files</string>
           </property>
          </widget>
         </item>
        </layout>
        <zorder>fileListWidget</zorder>
        <zorder></zorder>
        <zorder>clearAllPushButton</zorder>
        <zorder></zorder>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox_3">
        <property name="title">
         <string>Uniques</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_3" stretch="1,0,0,0,0">
         <item>
          <widget class="QListWidget" name="uniquesList"/>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="0,1">
           <item>
            <widget class="QLabel" name="label_3">
             <property name="text">
              <string>Count:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="uniqueLabel">
             <property name="text">
              <string>-</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_4" stretch="0,1">
           <item>
            <widget class="QLabel" name="label_4">
             <property name="text">
              <string>Similarity:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QDoubleSpinBox" name="similaritySpinBox">
             <property name="suffix">
              <string>%</string>
             </property>
             <property name="minimum">
              <double>10.000000000000000</double>
             </property>
             <property name="maximum">
              <double>99.000000000000000</double>
             </property>
             <property name="value">
              <double>95.000000000000000</double>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QCheckBox" name="useHeuristicCheckBox">
           <property name="text">
            <string>Use heuristic (more efficient find)</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="findButton">
           <property name="text">
            <string>Find</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QGroupBox" name="groupBox_2">
      <property name="title">
       <string>Preview:</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QTextEdit" name="previewEdit"/>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1015</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
